lesson_09/oop/oop_app/src/main/java/com/codedifferently/lesson9/richhawkins/Library.java

```diff
@@ -0,0 +1,54 @@
+package main.java.com.codedifferently.lesson9.richhawkins;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class Library {
+  private HashSet<Book> bookCollection = new HashSet<>();
```

Comment:
Prefer the interface `Set<Book>`.

https://stackoverflow.com/questions/2697783/what-does-program-to-interfaces-not-implementations-mean